src/.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*]

```

Comment:
The file in `src/.editorconfig` should be in `.editorconfig` (in the root of the project)